tests/sentry/processing/eventstore/test_processing.py

```diff
@@ -0,0 +1,16 @@
+from django.test import override_settings
+
+from sentry.eventstore.processing import event_processing_store, transaction_processing_store
+from sentry.testutils.pytest.fixtures import django_db_all
+from sentry.utils.services import LazyServiceWrapper
+
+
+@django_db_all
+@override_settings(
+    SENTRY_TRANSACTION_PROCESSING_STORE=None, SENTRY_TRANSACTION_PROCESSING_STORE_OPTIONS=None
```

Comment:
```suggestion
    SENTRY_TRANSACTION_PROCESSING_STORE=None, SENTRY_TRANSACTION_PROCESSING_STORE_OPTIONS={}
```

To better emulate the mistake from the previous attempt.